partials/instagram.php

```diff
@@ -1,12 +1,39 @@
-<?php 
+<section id="instagram-feed" class="row"> 
+
+<?php
 $instagram_handle = IGV_get_option( '_igv_instagram_handle' ); 
-?>
-<section class="row"> 
+if ($instagram_handle) {
+  //echo '@' . $instagram_handle;
+  // Get instagram feed
+  $instagram_feed = get_instagram_feed();
+
+  foreach($instagram_feed as $index => $instagram_item) {
+    $likes = $instagram_item->likes->count;
+    $comments = $instagram_item->comments->count;
+    $url = $instagram_item->images->standard_resolution->url;
+  ?>
+
+  <?php if( $index != 0 ) { ?>
+    <div class="col s1">&#8200;</div>
+  <?php } ?>
+
+  <div class="instagram-item col s4">
+    <a href="<?php echo $instagram_item->link; ?>">
+
+      <header>
+        <?php echo $likes ? $likes . '<span class="genericon genericon-heart"></span>' : ''; ?>
+        <?php echo $comments ? $comments . '<span class="genericon genericon-comment"></span>' : ''; ?>
+      </header>
+
+      <img src="<?php echo $instagram_item->images->standard_resolution->url; ?>" alt="" width="190px" />
```

Comment:
do you need to specify the width here? why not let `width: auto` deal with it? Also will it be worth at any point doing src set for this image?